x-pack/plugin/ent-search/src/main/java/org/elasticsearch/xpack/application/connector/ConnectorIndexService.java

```diff
@@ -87,7 +92,16 @@ public class ConnectorIndexService {
     // The client to interact with the system index (internal user).
     private final Client clientWithOrigin;
 
-    public static final String CONNECTOR_INDEX_NAME = ConnectorTemplateRegistry.CONNECTOR_INDEX_NAME_PATTERN;
+    private static final int CONNECTORS_INDEX_VERSION = 1;
+    public static final String CONNECTOR_INDEX_NAME = ".elastic-connectors";
```

Comment:
Nitpick: Rename to `CONNECTOR_ALIAS_NAME`

Reply:
I did, but `CONNECTOR_INDEX_NAME` was already used in a lot of places so I kept the name to reduce edits

Reply:
I've added a TODO to rename this to `CONNECTOR_ALIAS_NAME` in a follow up PR, just in the interest of keeping this PR less messy.